tests/integration/test_grading_run.py

```diff
@@ -170,3 +171,58 @@ def send_malformed_request(self):
             yield fetch(self.get_url('/api/v1/grading_run'), method='POST', headers=None, body='')
 
         self.assertTrue(400 <= context.exception.code <= 499)
+
+    @gen_test
+    def test_invalid_worker_register(self):
+        wr_invalid_token = '/api/v1/worker_register?token=ofsomething'
+        with self.assertRaises(tornado.httpclient.HTTPError) as context:
+            yield fetch(self.get_url(wr_invalid_token), method='GET',
+                    headers=None)
+        self.assertTrue(400 <= context.exception.code <= 499)
+
+    @gen_test
+    def test_missing_worker_register(self):
+        wr_missing_token = '/api/v1/worker_register'
+        with self.assertRaises(tornado.httpclient.HTTPError) as context:
+            yield fetch(self.get_url(wr_missing_token), method='GET',
+                    headers=None)
+        self.assertTrue(400 <= context.exception.code <= 499)
+
+    @gen_test
+    def test_invalid_worker_get_gj(self):
+        with self.assertRaises(tornado.httpclient.HTTPError) as context:
+            yield fetch(self.get_url('/api/v1/grading_run?worker_id=-1'), method='GET',
```

Comment:
`api/v1/grading_run` is an endpoint for clients and does not require worker id. Since you used method `GET` I assume you would be testing [this](https://github.com/illinois-cs241/broadway-api#get-apiv1grading_rungrading-run-id) endpoint. Here the grading run id is passed in the URL itself, not in parameters.